editors/vscode/client/extension.ts

```diff
@@ -251,3 +263,16 @@ export function deactivate(): Thenable<void> | undefined {
   }
   return client.stop();
 }
+
+function createFileEventWatchers(configRelativePath: string) {
+  const workspaceConfigPatterns = (workspace.workspaceFolders || []).map((workspaceFolder) =>
+    new RelativePattern(workspaceFolder, configRelativePath)
+  );
+
+  return [
+    workspace.createFileSystemWatcher('**/.oxlintrc.json'),
```

Comment:
should we extract the `.oxlintrc.json` part to a constant?

Reply:
I think it's only re-used in one time each in two different files, but yes I can refactor things to make it a constant and share it.